src/riak_kv_eleveldb_backend.erl

```diff
@@ -483,11 +483,12 @@ build_list({_K, _V}=KV, Acc) ->
 
 range_scan(FoldIndexFun, Buffer, Opts, #state{fold_opts=_FoldOpts,
                                               ref=Ref}) ->
-    {_, Bucket, Qry} = proplists:lookup(index, Opts),
+    {_, Bucket, Qry1} = proplists:lookup(index, Opts),
+    Qry2 = riak_kv_select:convert(riak_kv_select:current_version(), Qry1),
     ?SQL_SELECT{'WHERE'    = W,
                 helper_mod = _Mod,
                 local_key  =  #key_v1{ast = LKAST},
-                partition_key = #key_v1{ast = _PKAST}} = Qry,
+                partition_key = #key_v1{ast = _PKAST}} = Qry2,
```

Comment:
They are left over from a previous change, I have removed them in 98bb5103ecf482e75fda075e51254638e77cdb7a.